products.php

```diff
@@ -70,18 +44,488 @@
         'health_license_number' => '50/16098',
         'image'                 => 3,
         'is_public'             => 1,
-        'created_by'            => 1,
+        'created_by'            => 
     ],  [
-        'id'                    => 1,
-        'name'                  => '',
-        'alias'                 => '',
-        'description'           => '',
+        'id'                    => 4,
+        'name'                  => 'آبمیوه نکتار انبه',
+        'alias'                 => 'fruit juice mango nectar',
+        'description'           => 'آبمیوه   پوره طبعی انبه',
```

Comment:
```suggestion
        'description'           => 'آبمیوه پوره طبعی انبه',
```
چرا فضای خالی اضافی دارد؟